tests/EmitTest.elm

```diff
@@ -163,6 +162,28 @@ javascript =
                       )
                     ]
                 )
+
+            -- this [ _ , _ ] format will eventually have to change to something like List$fromArray([...]) when we add (::), (++), etc
+            , describe "List"
+                (List.map runTest
+                    [ ( "empty list"
+                      , List []
+                      , "[]"
+                      )
+                    , ( "single item in list"
+                      , List [ typed (Literal (Int 1)) ]
+                      , "[1]"
+                      )
+                    , ( "simple list"
+                      , List [ typed (Literal (Int 1)), typed (Literal (Int 2)), typed (Literal (Int 3)) ]
+                      , "[1, 2, 3]"
+                      )
+                    , ( "simple list2"
+                      , List [ typed (Literal (String "one")), typed (Literal (Int 2)), typed (Literal (Int 3)) ]
+                      , "[\"one\", 2, 3]"
```

Comment:
I'd drop this test as that source Expr is a bit nonsensical. Or let's change it to a list of three strings?